lib/kafka/consumer.rb

```diff
@@ -125,6 +132,9 @@ def subscribe(topic_or_regex, default_offset: nil, start_from_beginning: true, m
     def stop
       @running = false
       @fetcher.stop
+      unless @interceptors.nil?
+        @interceptors.close
```

Comment:
Should you run this in an ensure `block`?

If `@fetcher.stop` raises an exception, do you still need to `close`?

Reply:
closing the interceptors first since it doesn't throw exceptions